v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/uniformsplitter/range/PartitionColumnTest.java

```diff
@@ -74,7 +86,12 @@ public void testPartitionColumnPreconditions() {
             PartitionColumn.builder()
                 .setColumnName("col1")
                 .setColumnClass(String.class)
-                .setStringCollation("latin1_swedish_ci")
+                .setStringCollation(
+                    CollationReference.builder()
+                        .setDbCharacterSet("latin1")
+                        .setDbCollation("latin1_swedish_ci")
```

Comment:
Good to have -  tests from a different charset.

Reply:
This test is just testing the building for PartitionColumn, so here it won't matter.
But I can add another db-dump test for non utf8mb4 is needed.